sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/implementation/models/CommunicationErrorResponseException.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.administration.implementation.models;
```

Comment:
This should be generated inside the new `identity` package.

Reply:
removed

Reply:
removed